### PR TITLE
User prof ext2

### PR DESCRIPTION
**Pull request by niiita**
Made page links from menu bar and buttons. #49 
User profile page extension, front end work. #68
Default profile picture added. 
Edited buttons on answers page to be consistent with themes. #77

### CODE REVIEW

File: SA3/qa_web/templates/qa_web/UserProfile.html [hunk 1]
```diff
@@ -1,75 +1,83 @@
...
+<small>{{ vts.creation_date }}</small>
+{% endfor %}
 
 
-</head>
-</body>
+</div>
+
+<div class="col-sm-4">
+    likes n upvotes here?
```
Comment (alvyn279) [inline SA3/qa_web/templates/qa_web/UserProfile.html]: Let's make sure to get this done by Friday haha, or at least remove this text.

File: SA3/qa_web/templates/qa_web/UserProfile.html [hunk 2]
```diff
@@ -1,75 +1,83 @@
...
+        <p class="info">City: {{ displayed_user.city }}</p>
 
-button:hover, a:hover {
-  opacity: 0.7;
-}
-</style>
-</head>
-<body>
+        <a href="#">
+            <button>Click for Home Page</button>
```
Comment (alvyn279) [inline SA3/qa_web/templates/qa_web/UserProfile.html]: `href` needs to be populated to home page: `'/'`

File: SA3/qa_web/templates/qa_web/UserProfile.html [hunk 3]
```diff
@@ -1,75 +1,83 @@
...
-  width: 100%;
-  font-size: 18px;
-}
+{% block styles %}
+<link href="{% static 'css/qa.css' %}" rel="stylesheet"/>
+<link rel="stylesheet" href="{% static 'css/one-page-wonder.min.css' %}"/>
+<link rel="stylesheet" href="{% static 'css/pagedown.css' %}"/>
+<link rel="stylesheet" href="{% static 'css/base.css' %}"/>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+{% endblock %}
```
Comment (alvyn279) [inline SA3/qa_web/templates/qa_web/UserProfile.html]: Nice!

File: SA3/qa_web/templates/qa_web/UserProfile.html [hunk 4]
```diff
@@ -1,75 +1,83 @@
...
-button:hover, a:hover {
-  opacity: 0.7;
-}
-</style>
-</head>
-<body>
+        <a href="#">
+            <button>Click for Home Page</button>
+        </a>
+        <button>EDIT MY PROFILE</button>
```
Comment (Lercerss) [inline SA3/qa_web/templates/qa_web/UserProfile.html]: Not linked to anything?

File: SA3/qa_web/templates/qa_web/includes/navbar.html [hunk 5]
```diff
@@ -12,6 +17,9 @@
                         {# A link to the user profile should be added here #}
                         <a class="nav-link" href="#">Welcome {{ user.username }}</a>
```
Comment (Lercerss) [inline SA3/qa_web/templates/qa_web/includes/navbar.html]: Maybe we could link to the user's profile here?

**Pull request merged**